bitnami/spark/templates/scv-master.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
```

Comment:
I think the filename was meant to be svc-master.yaml, not scv-master.yaml

Reply:
Changed.